src/pcp/atop/photoproc.c

```diff
@@ -365,6 +413,8 @@ photoproc(struct tstat **tasks, unsigned long *taskslen)
 		update_task(&(*tasks)[i], pids[i], insts[i], result, descs, offset);
 	}
 
+	cpu_accumulation(tasks, count);
```

Comment:
Suggest renaming this to 'thread_cpu_accumulation' so its more clear what it does and that its associated with the qsort callback which is named that way.